happiness-backend/api/models/schema.py

```diff
@@ -108,6 +108,10 @@ class Meta:
     timestamp = ma.Str(dump_only=True)
 
 
+class CommentEditSchema(ma.Schema):
+    data = ma.Str(required=True)
+
+
```

Comment:
Could we make a more generalized schema for a single string data so we don't have to keep adding new schemas like this?